src/js/containers/router/RouterRoutes.jsx

```diff
@@ -226,6 +226,15 @@ const routeDefinitions = {
             }
         },
         {
+            path: 'technical-history',
```

Comment:
If you look at our other paths, we tend to do camelCase rather than dashes between words. It'd be best to be consistent.